docs/docs-content/clusters/data-center/nutanix/nutanix.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_label: "Nutanix"
+title: "Nutanix"
+description: "Learn how to configure Nutanix and create Nutanix clusters in Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["data center", "nutanix"]
+---
+
+
+[Nutanix](https://www.nutanix.com/what-we-do) is a hyper-converged infrastructure platform that combines storage, compute, and networking into a single integrated system. Palette provides a generic framework built upon the open-source [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io) initiative to support adding cloud providers. As a Technical Preview of implementing this framework, you can deploy Kubernetes clusters on Nutanix using Palette.  Integration with Nutanix is achieved through a Private Cloud Gateway (PCG), which establishes a secure connection with Nutanix.
+
+## Get Started
+
+Once a system administrator registers Nutanix with Palette, the PCG is installed, and your Nutanix cloud account is added, you can deploy a cluster to Nutanix by using Palette. Check out the [Deploy a Cluster with Palette](../../public-cloud/deploy-k8s-cluster.md) tutorial to get started.
+
+The section below describes the workflow to register Nutanix with Palette and deploy a cluster. 
+
+## Workflow
+
+The following summary steps outline the CAPI cloud workflow illustrated in the diagram. Since only self-hosted instances of Palette have access to the system console by a system administrator, the registration step is performed at the system level, making the cloud provider available to all the tenants. CAPI users perform the remaining steps in Palette and their local environment.
```

Comment:
```suggestion
The following summary steps outline the Nutanix cloud workflow illustrated in the diagram. Since only self-hosted instances of Palette have access to the system console by a system administrator, the registration step is performed at the system level, making the Nutanix cloud available to all the tenants. Regular Palette users perform the remaining steps in Palette and their local environment.
```